### ISSUE TITLE
Please explain plugin usefulness

### ISSUE THREAD

**Issue by pcav** [lequynhhoa/Style-NFI-Map#1]
What is the advantage of this approach in comparison to the much simpler providing of a sample project and / or set of standard styles?
Thanks.

**Comment by lequynhhoa** [comment]
The advantage of this approach is quickly style that I make style, QGIS have not avaiable. 

For example: Forest layer, we have 93 code style for Viet Nam, each province have other style (in 93 code).
With this plugin can be get code and style color.

**Comment by pcav** [comment]
Sorry, I'm not sure I understand you.

**Comment by lequynhhoa** [comment]
As I say, it makes style color quicker. For example: the user no need edit line, border line, partern...

**Comment by pcav** [comment]
In what sense this is different from providing a default style?

**Comment by lequynhhoa** [comment]
Yes. But only use for NFI Map (National Forest Map) and other map can't use this style.
This plugin can change label based on fieldname (code).
For example:
![image](https://user-images.githubusercontent.com/4716948/29061453-301813f6-7c48-11e7-92a0-fb38ae641bca.png)

**Comment by pcav** [comment]
Sorry, I still fail to understand. Could you provide sample data so I can test and probably understand as I see it in action?
Thanks.

**Comment by lequynhhoa** [comment]
Yes :(. This is my data : https://drive.google.com/open?id=0Bx55JsBuQY73d2Mta1ByVERxX3c

**Comment by pcav** [comment]
I cannot make the plugin work, see #2 
Anyway, it seems to me that what the plugin does is simply applying a style to a layer: am I wrong?

**Comment by lequynhhoa** [comment]
Sorry pcav. A long time, I just come back with this plugin.
#2 what you said, you need to use extention "_rungkk" file with the first tool.
You can see my video: https://youtu.be/nfV6U-scFn4.

Could you approve it, I want to share it for my community in Vietnam, they can use it.
If it has other problems, I will fix it. I need them check for more their layer

Thank you

**Comment by NyakudyaA** [comment]
@pcav  I have looked at the plugin and the code. I think this plugin is basically taking a column and applying a style. In this case, the style is defined as a categorized renderer in the code.  I am not sure if it makes sense to publish this plugin because as much as it might make it easier for people with this kind of data to apply style I think it does not offer a functionality that enhances QGIS. 

How about creating a repository that contains all the styles you have specified and the people can choose to use the built-in functionality to import a style and they could also use the inbuilt function to have multiple styles per layer. Please see the following [Style sheets](https://github.com/charleyglynn/OSM-Shapefile-QGIS-stylesheets). 

@lequynhhoa How about enhancing the plugin to fetch the data from the custodians using an API and then dynamically apply the styles when you download the layers into QGIS.

**Comment by lequynhhoa** [comment]
@NyakudyaA Yes. The plugin is basically taking a column and applying a style for layer that is regulated in Forestry of VietNam. This style is defined by a catagorized. The user can update style easier. In fact, using .qml also fastly but in this case, I want to use this plugin.
